octobot_trading/personal_data/orders/channel/orders_updater.py

```diff
@@ -98,11 +98,12 @@ async def _open_orders_fetch_and_push(self, is_from_bot=True, limit=ORDERS_UPDAT
             open_orders: list = await self.channel.exchange_manager.exchange.get_open_orders(symbol=symbol, limit=limit)
             if open_orders:
                 await self.push(orders=list(map(self.channel.exchange_manager.exchange.clean_order, open_orders)),
-                                is_from_bot=is_from_bot)
+                                is_from_bot=is_from_bot,
+                                is_from_init=is_from_init)
             else:
                 await self.handle_post_open_order_update(symbol, open_orders, False)
 
-    async def _closed_orders_fetch_and_push(self, limit=ORDERS_UPDATE_LIMIT) -> None:
+    async def _closed_orders_fetch_and_push(self, limit=ORDERS_UPDATE_LIMIT, is_from_init=False) -> None:
```

Comment:
Why ?

Reply:
to be able to forward the info in orders channel